dart/lib/src/client_reports/outcome.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:meta/meta.dart';
+
+@internal
+enum Outcome { ratelimitBackoff, networkError }
```

Comment:
`m`: On Cocoa and .NET, we named this DiscardReason, as in the protocol the field is `reason`.